types/chai/index.d.ts

```diff
@@ -663,11 +663,10 @@ declare namespace Chai {
         /**
          * Asserts that value is not undefined.
          *
-         * @type T   Type of value.
          * @param value   Actual value.
          * @param message   Message to display on error.
          */
-        isDefined<T>(value: T, message?: string): void;
+        isDefined(value: any, message?: string): asserts value;
```

Comment:
As an alternative to `any`, would prefer either `unknown` or leaving as a type parameter.

Reply:
i tried fix the build error. 
```
Assertions require every name in the call target to be declared with an explicit type annotation.ts(2775)
chai-tests.ts(5, 7): 'assert' needs an explicit type annotation.
```
but its not about the generic of the method.

Reply:
in addition its not possible to raise the typescript version without modifying other packages